src/utilities/fe_mesh.jl

```diff
@@ -0,0 +1,162 @@
+# Typ definitions
+type FEBoundary # Tänkte typ parametrisera detta på nåt sätt så att man har FEboundary i 2D, 1D, och 0D
```

Comment:
Kan man göra det typ:
```jl
type FEBoundary{T}
    topology::Array{Int}
end
```

and then for a 2D square mesh generate 4 `FEBoundary{1}` for the 1D line boundaries and 4 `FEBoundary{0}` for the 0D corners?